Source/WebKit/WebProcess/WebPage/mac/WKAccessibilityWebPageObjectBase.mm

```diff
@@ -62,10 +62,8 @@ @implementation WKAccessibilityWebPageObjectBase
     if (auto* localMainFrame = dynamicDowncast<WebCore::LocalFrame>(page->mainFrame())) {
         if (auto* document = localMainFrame->document())
             return document->axObjectCache();
-    } else if (auto* remoteMainFrame = dynamicDowncast<WebCore::RemoteFrame>(page->mainFrame())) {
-        auto& tree = remoteMainFrame->tree();
-        auto* firstFrame = dynamicDowncast<WebCore::LocalFrame>(tree.firstChild());
-        auto* document = firstFrame ? firstFrame->document() : nullptr;
+    } else if (auto remoteLocalFrame = self.remoteLocalFrame) {
```

Comment:
Since `remoteLocalFrame` returns a pointer type, we should specify it instead of using `auto` (ideally we'd use a `RefPtr` if possible)